get-latest-tgf.ps1

```diff
@@ -0,0 +1,40 @@
+$ErrorActionPreference = "Stop" #Make all errors terminating
+
+try {
+    $TGF_PATH = (Get-Command tgf.exe).Path
+    Write-Host tgf path: $TGF_PATH
+    $LOCAL_VERSION = (tgf.exe --current-version) | Out-String
+    $LOCAL_VERSION = [regex]::Matches($LOCAL_VERSION,'\d+\.\d+\.\d+').Value
+} catch {
+    $TGF_PATH = Join-Path -Path (Get-Location) -ChildPath "tgf.exe"
+    Write-Host tgf not found. using $TGF_PATH
+} finally {
+    Write-Host "- tgf version (local):" $LOCAL_VERSION
+}
+
+try {
+    $bVersion = (Invoke-WebRequest -Uri "https://coveo-bootstrap-us-east-1.s3.amazonaws.com/tgf_version.txt").Content
```

Comment:
I am not sure that it is useful to keep that mechanism. You can directly checkout the latest version and if you already have tgf installed, it should update itself without having to rely on a script.